<ui version="4.0" >
 <class>EmuSettingsClass</class>
 <widget class="QMainWindow" name="EmuSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="fileButton" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>180</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlsButton" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>260</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>Controls</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>260</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audioButton" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>260</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinueButton" >
    <property name="geometry" >
     <rect>
      <x>510</x>
      <y>180</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QWidget" native="1" name="settingsViewWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>150</height>
     </rect>
    </property>
    <widget class="QLabel" name="atntLabel" >
     <property name="geometry" >
      <rect>
       <x>-1</x>
       <y>0</y>
       <width>641</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton" >
     <property name="geometry" >
      <rect>
       <x>510</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text" >
      <string>About</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="saveloadState" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>150</y>
      <width>241</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string notr="true" >QGroupBox{
border-color: rgb(0, 0, 0);
background-color: rgb(192, 192, 192 );
}</string>
    </property>
    <property name="title" >
     <string>Save/Load State</string>
    </property>
    <widget class="QPushButton" name="saveStateButton" >
     <property name="geometry" >
      <rect>
       <x>158</x>
       <y>40</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
    <widget class="QComboBox" name="saveSlotBox" >
     <property name="geometry" >
      <rect>
       <x>84</x>
       <y>40</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <item>
      <property name="text" >
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="loadStateButton" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>40</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text" >
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="videoButton" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>260</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton" >
    <property name="geometry" >
     <rect>
      <x>380</x>
      <y>180</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text" >
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="atnresources.qrc" />
 </resources>
 <connections/>
</ui>
